tests/mocha/variant_browser.js

```diff
@@ -92,6 +98,30 @@ function variantSearchBySpeciesandChrLocation(driver){
         });
     });
 }
+
+function positionFilterBoxValidation(driver){
+    driver.findElement(By.id("selectFilter-trigger-picker")).click();
+    driver.findElement(By.xpath("//li[text()='Chromosomal Location']")).click();
+    driver.findElement(By.id("speciesFilter-trigger-picker")).click();
+    driver.findElement(By.xpath("//li[text()='Mosquito / AaegL3']")).click();
+    driver.findElement(By.name("region")).clear();
+    driver.findElement(By.name("region")).sendKeys('supercont1.18:165624-165624');
```

Comment:
for
letter: X:10000000-11000000
digits: 2:4000000-4100000
alphanumeric: supercont1.18:165624-165624 are added.
do you know a example for chr with different punctuation?.